Kiosk/App/AppDelegate.swift

```diff
@@ -40,7 +48,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         defaults.removeObject(forKey: XAppToken.DefaultsKeys.TokenExpiry.rawValue)
 
         let auctionStoryboard = UIStoryboard.auction()
-        let appViewController = auctionStoryboard.instantiateInitialViewController() as? AppViewController
+        let appViewController = auctionStoryboard.instantiateInitialViewController()
+            as? AppViewController
```

Comment:
How it breaks this line seems really strange to me.

Reply:
Yeah, the breakpoint in front of `as?` was rather an experiment. Would you rather have it break after `as?` (instead of front), after `=`, or after `auctionStoryboard.`?

Reply:
Hmm! I think breaking after `=` seems the least weird 😅